src/pages/api/users/index.js

```diff
@@ -16,6 +18,53 @@ export default async function handler(req, res) {
         message: e.message,
       });
     }
+  } else if (req.method == "POST") {
```

Comment:
Only "Admin" and "Manager" roles can invite users, so make sure to check the session here.
You can use NextAuth's `getServerSideSession()`, which will give you `session.user.role`

Reply:
Done; I think this also needs to be implemented for other backend functions as well